lib/dataClasses/seed_response.dart

```diff
@@ -0,0 +1,43 @@
+import 'dart:convert';
+
+/// The SeedResponse class represents the data from a response from the seed
+/// server. It accepts input for deserialization. The factory constructors will
+/// not pass empty or default versions of the class. Rather, they throw
+/// expcetions if the data passed in are invalid.
+class SeedResponse {
```

Comment:
**question:** could this class be immutable? What are the benefits or drawbacks if it was able to be?

Reply:
It could, but I feel it would do more harm than good. I regard this class to be a 'slice in time'. If we start imputing the values received from the server, we'll stop respecting the spirit of the server's response.